src/js/components/Diagram/stories/Animated.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Diagram, Grommet, grommet, Stack, Text } from 'grommet';
+import { Diamond } from 'grommet-icons';
+import { deepMerge } from 'grommet/utils';
+
+import { data } from './data';
+
+const customTheme = deepMerge(grommet, {
+  diagram: {
+    extend:
+      '@keyframes example { to { stroke-dashoffset: 0; } } path { stroke-dasharray: 500; stroke-dashoffset: 500; animation: example 3s linear forwards; }',
```

Comment:
I wonder if we should build this into Diagram? We could add `animation: true|false|draw|pulse` to both Diagram as a whole but also to individual objects in `connections`.

Reply:
I love it. 
Do you think it can be on a separate PR?

Reply:
sounds like a separate addition :)

Reply:
#3210